src/ansys/fluent/core/session_solver.py

```diff
@@ -77,16 +77,17 @@ class Solver(BaseSession):
     def __init__(
         self,
         fluent_connection,
-        remote_file_handler: Optional[Any] = None,
+        file_transfer_service: Optional[Any] = None,
```

Comment:
This will need a Fluent-side change. I was passing a PimFileTransferService instance as remote_file_handler due to a previous issue, now we can remove that keyword argument from Fluent code.

Reply:
@mkundu1 Done. Thank you. !469694, t994337